com.unity.render-pipelines.high-definition/Documentation~/Render-Pipeline-Debug-Window.md

```diff
@@ -40,6 +41,20 @@ To display the current active item independently of the debug window:
 - **Xbox controller**: Press the X button.
 - **PlayStation controller**: Press the Square button.
 
+<a name="RenderGraphPanel"></a>
+
+## Render Graph panel
+
+The **Render Graph** panel has tools that you can use to debug the [Render Graph](https://docs.unity3d.com/2020.2/Documentation/Manual/render-graph.html) used by HDRP.
+
+| **Debug Option**  | **Description**                                              |
+| ----------------- | ------------------------------------------------------------ |
+| **Clear Render Targets at creation**  | Enable the checkbox to clear render targets the first time they are used. |
```

Comment:
Enable the checkbox to make the render graph system clear render targets the first time it uses them.